src/components/Tooltip/index.js

```diff
@@ -106,8 +106,8 @@ const Tooltip = ({
       <TooltipReference {...tooltip} ref={children.ref}>
         {finalChildren}
       </TooltipReference>
-      <ReakitTooltip {...tooltip}>
-        <StyledTooltip variant={variant} {...props}>
+      <ReakitTooltip {...tooltip} {...props}>
```

Comment:
As StyledTooltip is a Box onto which we pass styled props this will break styling no ? 

We could extract zIndex from props and only pass that to the reakit tooltip ?

Reply:
@chambo-e It now should be better